From 5f582a7b1fe65bbbb0cc817e6b9cd4310afc1218 Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew.moseley@northern.tech>
Date: Sun, 15 Jul 2018 15:52:34 -0400
Subject: [PATCH 1/2] rtl8723bs: Disable CONFIG_DEBUG

Signed-off-by: Drew Moseley <drew.moseley@northern.tech>
---
 include/autoconf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/autoconf.h b/include/autoconf.h
index 9aa596f..d1a33fe 100755
--- a/include/autoconf.h
+++ b/include/autoconf.h
@@ -286,7 +286,7 @@
 /*
  * Debug Related Config
  */
-#define CONFIG_DEBUG
+/* #define CONFIG_DEBUG */
 
 #ifdef CONFIG_DEBUG
 #define DBG	1	// for ODM & BTCOEX debug
-- 
2.7.4

